### DISCUSSION TITLE
Open app directly to database

### DISCUSSION THREAD

**Discussion by uggrin** [Kunzisoft/KeePassDX#1020] [Ideas]
Hi, I've been using this app for a while now, and I find it great.

But i use only one database to store my passwords, so it would be very convenient if there were an option to open the app directly to the enter master password screen.

Thanks!

**Comment by J-Jamet** [comment]
Simply select the star next to the base to bookmark it.

**Reply by J-Jamet** [reply]
But in this case, the entry selection workflow will also go directly through the master password screen (and not through the base selection).

**Reply by uggrin** [reply]
> Simply select the star next to the base to bookmark it.

You know you'd think I'd try that before posting here. Thanks!

Also if i tap back on that screen it does go through the database selection screen.

**Reply by J-Jamet** [reply]
You're welcome.

> Also if i tap back on that screen it does go through the database selection screen.

Yes it is normal
